server/routes/students.js

```diff
@@ -55,4 +55,14 @@ router.post('/', async (req, res, next) => {
     }
 });
 
+router.post('/one', async (req, res, next) => {
```

Comment:
Can we change this method to `GET` and have the path be `/:email` so that we can read `email` from `req.params.email`

Reply:
I hesitate to do this because it leaves PII (the email) in the logs on the client and server side in plaintext, whereas `POST` does not

This is outlined [here](https://security.stackexchange.com/a/29605), and can be seen in the dev console.

This is with `POST`:
<img width="763" alt="Screen Shot 2020-05-24 at 6 16 21 PM" src="https://user-images.githubusercontent.com/54961598/82766184-f5cd4500-9dea-11ea-9424-ef4771fff886.png">

This is with `GET`:
<img width="526" alt="Screen Shot 2020-05-24 at 6 17 01 PM" src="https://user-images.githubusercontent.com/54961598/82766208-10072300-9deb-11ea-9e55-3650ca916185.png">